samples/java-android-app/src/main/java/com/amplitude/android/sample/MainApplication.java

```diff
@@ -84,4 +84,8 @@ public BaseEvent execute(@NonNull BaseEvent event) {
         event.getEventProperties().put("custom android event property", "test");
         return event;
     }
+
+    @Override
```

Comment:
@Mercy811 I thought this wouldn't be necessary with the `-Xjvm-default=all` change. Is there something we can do to prevent this being required?

Reply:
it's the replacement for `JvmDefault` not `@Override` unfortunately. https://blog.jetbrains.com/kotlin/2020/07/kotlin-1-4-m3-generating-default-methods-in-interfaces/#jvmdefaultwithoutcompatibility

Reply:
It was throwing a compilation error when I tried running the app locally. Do we add -Xjvm-default=all to our config while running it?

Reply:
Yes. It's in [build.gradle](https://github.com/amplitude/Amplitude-Kotlin/blob/1ee97e60cd7859755232c3bc7f05693ba6716561/build.gradle#L31). Could you sync gradle?

Reply:
Thanks for the help @Mercy811. The gradle sync helped remove the error.